lttng-relayd.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
How many services is this project deliver to the system?
If it's more than one service you should update the artifacts.yaml to reflect that.